src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsHopObject.java

```diff
@@ -22,8 +23,8 @@
 
   private static final String POST_MESSAGE_SCRIPT_XPATH =
       "//script[contains(text(), 'parent.postMessage')]";
-
-  private final static ImmutableMap<String, String> dfpSrc =
+  private static final int AD_SUCCESS_TIMEOUT_SEC = 15;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)